scripts/zones/Apollyon/mobs/Gunpod.lua

```diff
@@ -2,16 +2,100 @@
 -- Area: Apollyon Central
 --  Mob: Gunpod
 -----------------------------------
-require("scripts/globals/limbus");
------------------------------------
+local loot = 
+{
+    [1] = -- AF
+    {
+        {
+            {itemid = 1933, droprate =  9}, -- MNK
+            {itemid = 1931, droprate = 53}, -- WAR
+            {itemid = 1959, droprate =  6}, -- SMN
+            {itemid = 1935, droprate = 12}, -- WHM
+            {itemid = 1945, droprate = 29}, -- DRK
+            {itemid = 1957, droprate = 12}, -- DRG
+            {itemid = 1949, droprate = 35}, -- BRD
+            {itemid = 2659, droprate = 35}, -- COR
+            {itemid = 1939, droprate = 12}, -- RDM
+            {itemid = 1951, droprate = 12}, -- RNG
+            {itemid = 2661, droprate = 12}, -- PUP
+            {itemid = 1937, droprate = 18}, -- BLM
+            {itemid = 1955, droprate = 29}, -- NIN
+            {itemid = 2717, droprate = 12}, -- SCH
+            {itemid = 1947, droprate = 12}, -- BST
+            {itemid = 2657, droprate = 18}, -- BLU
+            {itemid = 2715, droprate =  5}, -- DNC
+            {itemid = 1953, droprate = 35}, -- SAM
+            {itemid = 1941, droprate = 41}, -- THF
+            {itemid = 1943, droprate = 18}, -- PLD
+        },
+    },
+    [2] = -- Chip
+    {
+        {
+            {itemid = 1987, droprate = 53}, -- Charcoal Chip
+            {itemid = 1988, droprate = 76}, -- Magenta Chip
+            {itemid = 1909, droprate = 64}, -- Smalt Chip
+            {itemid = 1910, droprate = 41}, -- Smoky Chip
```

Comment:
Looking at the [spiffy new chart SE has provided us](https://www.bg-wiki.com/bg/Treasure_Hunter), I think these might all be in the Rare category.

But I don't know how these rates apply to the drop groups that gunpods have - could it be that an entire category has a certain rate (ie: Very Common or Common), and then drops within that category are evenly distributed?

Reply:
I figured the Gunpod is similar spread as the chests. I was thinking they're probably all even chance but just went with wiki drop rates since that was what everything else was using, and set each group as an equal chance of dropping from the Gunpod.